HelloWars/Arena/ViewModels/MainWindowViewModel.cs

```diff
@@ -67,20 +76,52 @@ public ICommand AutoPlayCommand
             get { return _autoPlayCommand ?? (_autoPlayCommand = new AutoPlayCommand(this)); }
         }
 
+        public ICommand OnLoadedCommand
+        {
+            get { return _onLoadedCommand ?? (_onLoadedCommand = new CommandBase(OnLoaded())); }
+        }
+
+        private Predicate<object> OnLoaded()
+        {
+            Game = _gamePlugins.FirstOrDefault(f => (f.GetType().Name == _arenaConfiguration.GameType));
```

Comment:
Has to be better way. It creates instances of every game despite us playing only one during whole application run.